README.md

```diff
@@ -50,7 +71,9 @@ func init() {
 				Panic: 13631488,
 				Fatal: 13631488,
 			},
-			DisableInlineFields: false, // If set to true, fields will not appear in columns ("inline")
+			DisableTimestamp: false,           // Setting this to true will disable timestamps from appearing in the footer
+			TimestampFormat:  timeStampFormat, // The timestamp takes this format; if it is unset, it will take logrus' default format
+			TimestampLocale:  nil,             // nil == time.Local, time.UTC, time.LoadLocation("America/New_York"), etc
```

Comment:
I'd prefer the comment `// The timestamp uses this locale; if it is unset, it will use time.Local`. By convention all usage has been left until later in the README.

Reply:
✔